core/src/main/java/com/scalar/db/transaction/consensuscommit/Snapshot.java

```diff
@@ -64,6 +64,9 @@ public class Snapshot {
   private final Map<Key, Put> writeSet;
   private final Map<Key, Delete> deleteSet;
 
+  // The scanner set used to store information about scanners that are not fully scanned
+  private final List<ScannerInfo> scannerSet;
```

Comment:
Added scannerSet that's used to store information about scanners that are not fully scanned.